### DISCUSSION TITLE
How do I make an “author” or “contributor” appear in a book chapter?

### DISCUSSION THREAD

**Discussion by argideritzalpea** [quarto-dev/quarto-cli#6535] [Q&A]
### Description

In my book `_quarto.qmd`, I am specifying a particular chapter for which I want to ensure a “contributor” field appears underneath the title. At the top of the qmd file is the following:
```
title: XYZ
contributor: Jake
```
However, the rendered HTML does not show the contributor at all. 

How do I ensure that it is shown in the HTML rendered document?

**Comment by mcanouil** [comment]
`contributor` has no meaning unless you give one using a LUA filter or the meta shortcode.
See:
- https://quarto.org/docs/extensions/filters.html
- https://quarto.org/docs/authoring/variables.html#meta

What does have a meaning is `author`, see <https://quarto.org/docs/journals/authors.html#author-schema>.

In the end, I am not sure exactly what you want to do ...

**Reply by renatovargas** [reply]
I am having the same issue. I believe what he means is that for a book which is a collection of contributions, you want to set authorship per chapter, not for the whole document as the docs suggest and have it show that way too in the final output next to the title of the chapter.  It is common in scientific article compilation books or institutional report chapters.

Chapter 1 by Author A.
Chapter 2 by Author B.
Chapter 3 by Authors A and B.
Chapter 4 by Author A.
Chapter 5 by Author C.

**Reply by novica** [reply]
Replying here because I am interested in the same question.

**Reply by mcanouil** [reply]
Still, the answer is the one given.
Something a bit similar to https://github.com/pandoc-ext/section-bibliographies but to add authors.
